### PR TITLE
[NT-1944] UI - User badges

### PR DESCRIPTION
**Pull request by moderateepheezy**
# 📲 What

Used the same author badge style for `Collaborator`s and `Creator`s.

# 🤔 Why

Every author of a comment should have a badge except if the author of the comment is a backer. However, we are currently not showing a badge for collaborators. `Creator`s and `Collaborator`s are more like, we want to also add a badge to comments from a collaborator, the style of the badge for a `Collaborator` should be the same as the one for a `Creator`.

# 🛠 How

More in-depth discussion of the change or implementation.

# 👀 See

Trello, screenshots, external resources?

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# ✅ Acceptance criteria

- [ ]  Users can clearly identify posts by creators/collaborators, superbackers, and their own

For Collaborator, test with project https://staging.kickstarter.com/projects/paulomcg/the-project

### CODE REVIEW

File: Library/Styles/CommentCellStyles.swift [hunk 1]
```diff
@@ -27,6 +27,18 @@ public let commentBodyTextViewStyle: TextViewStyle = { textView in
     |> \.dataDetectorTypes .~ .link
 }
 
+// Internationalize
+
+public let collaboratorAuthorBadgeStyle: PaddingLabelStyle = { label in
```
Comment (singhhari) [inline Library/Styles/CommentCellStyles.swift:32]: Can we pass the text as an argument and eliminate the need for a separate style here?

**Review by singhhari — CHANGES_REQUESTED** [review]
working as expected. left a cpl of small comments.

File: Library/Styles/CommentCellStyles.swift [hunk 2]
```diff
@@ -27,6 +27,18 @@ public let commentBodyTextViewStyle: TextViewStyle = { textView in
...
+  label
+    |> \.text .~ localizedString(key: "Collaborator", defaultValue: "Collaborator")
```
Comment (msadoon) [inline Library/Styles/CommentCellStyles.swift:34]: Do we not have internationalized `Strings.` for this? If not, please add a `TODO: Internationalize`

**Review by msadoon — APPROVED** [review]
good to go!
One small thing, in `CommentCellHeaderStackViewTests` we might in the future want to test all localized strings for these badges to ensure there are no cut-offs.

File: Library/Styles/CommentCellStyles.swift (hunk 2 above)
Comment (moderateepheezy) [reply Library/Styles/CommentCellStyles.swift:34]: No, I couldn't fine one.

**Review by singhhari — APPROVED** [review]
lgtm

**Pull request merged**